test/test_ensemble_kalman_inversion.jl

```diff
@@ -104,7 +104,7 @@ architecture = CPU()
         @testset "PseudoSteppingSchemes tests" begin
             @info "  Testing pseudo-stepping schemes with default hyperparameters"
 
-            for pseudo_stepping in [ConstantPseudoTimeStep, Default, Chada2021, ConstantConvergence, Kovachki2018, Kovachki2018InitialConvergenceRatio]
+            for pseudo_stepping in [ConstantPseudoTimeStep, Default, Chada2021, ConstantConvergence, Kovachki2018, Kovachki2018InitialConvergenceRatio, Iglesias2021]
```

Comment:
What's "Default"?

Reply:
Do we need `ConstantPseudoTimeStep`, or would it be simpler to just interpret a `Number` as being a constant time-step?

Reply:
Well, I guess `pseudo_stepping=0.1` isn't so clear

Reply:
so forget that.

Reply:
I think previously we had an API that interpreted `pseudo_stepping=nothing` as "don't change the time-step". Then if the user wanted to set a time-step, they would do so by setting `pseudo_dt`.

Is this PR proposing to change the API?

Reply:
It was the default used in EnsembleKalmanProcesses.jl. I could change the name to `ThresholdedConvergenceRatio` or `MinimumConvergenceRatio`.